libs/designer/src/lib/ui/panel/templatePanel/createWorkflowPanel/tabs/reviewCreateTab.tsx

```diff
@@ -75,8 +88,127 @@ export const ReviewCreatePanel = () => {
       id: 'OaUode',
       description: 'Accessibility label for no configuration required',
     }),
+    // Compact view strings
+    ACTION: intl.formatMessage({
+      defaultMessage: 'Action',
+      id: 'F9hn5p',
+      description: 'Label for action description',
+    }),
+    ACTION_NAME: intl.formatMessage({
+      defaultMessage: 'Your action name',
+      id: '4ZDZFr',
+      description: 'Label for action name',
+    }),
+    SUBSCRIPTION: intl.formatMessage({
+      defaultMessage: 'Subscription',
+      id: 'X/pilm',
+      description: 'Label for subscription',
+    }),
+    LOCATION: intl.formatMessage({
+      defaultMessage: 'Location',
+      id: 'OO37qq',
+      description: 'Label for location',
+    }),
+    RESOURCE_GROUP: intl.formatMessage({
+      defaultMessage: 'Resource group',
+      id: 'YAGa7I',
+      description: 'Label for resource group',
+    }),
```

Comment:
We already have these resources used in resourcePicker, so lets reuse those resource strings.

Reply:
What do you mean by reuse the resource strings?  In resource picker they're inside a useMemo that uses an instance of the intl hook, that's not really something we can export afaik.  And it was my understanding that we can't pull out the objects from the formatMessage calls since intl won't pick those up when generating it's translation files.

Reply:
We could probably try making a hook to just hold a big object of all of our strings if we want to go that route.